www/templates/diary/list.html

```diff
@@ -49,7 +49,7 @@
                 </div>
 
 
-                <div id="diary-card" ng-class="tripgj.listCardClass" ng-style="{height: '335px'}">
+              <div id="diary-card" ng-class="tripgj.listCardClass" ng-style="{height: '315px'}">
```

Comment:
What is wrong with the previous size?

Reply:
We get rid of the label on top and simply display button. So I adjust the height of the container to match.
<img width="242" alt="Screen Shot 2019-05-25 at 21 25 22" src="https://user-images.githubusercontent.com/5247367/58397621-00d88280-807c-11e9-8013-f49826143dfd.png">